libs/web/hooks/src/useProposalNewForm.ts

```diff
@@ -13,7 +13,7 @@ import {
 import { useCENNZApi, useCENNZWallet } from "@app-gov/web/providers";
 
 export interface ProposalNewFormState {
-	step?: "Await" | "Submit" | "Process";
+	step: "Idle" | "Await" | "Submit" | "Process" | "Success";
```

Comment:
Why do we need a `"Success"` state now?

Reply:
To make the logic for `StepProgress` more simple & accurate - it can now check against the 4 states and match directly to the displayed text - 
```javascript
stepIndex=["Await", "Submit", "Process", "Success"].indexOf(formState.step) 
```